Dockerfile.downstream

```diff
@@ -1,27 +1,27 @@
+ARG TARGETARCH
+
 # Build the manager binary
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
```

Comment:
don't u need --platform=linux/$TARGETARCH here as well ?

Reply:
I added it, thanks!